### DISCUSSION TITLE
按键翻页工作有点问题

### DISCUSSION THREAD

**Discussion by HofNature** [xiaoyaocz/flutter_dmzj#144] [General]
现在每次打开漫画之后，直接按PageUP和PageDown是不能翻页的，但是按<-和->键可以翻页，而且只要按过一次<-或->进行翻页之后，PageUP和PageDown也就能翻页了，非常奇怪。

**Comment by HofNature** [comment]
另外用按键跨章翻页时动画有点问题

**Comment by xiaoyaocz** [comment]
https://github.com/xiaoyaocz/flutter_dmzj/commit/1423f5ede383bac774799bcff02157b24e59fe1a
